docs/outlook/item-multi-select.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Activate your Outlook add-in on multiple messages (preview)
+description: Learn how to activate your Outlook add-in when multiple messages are selected.
+ms.topic: article
+ms.date: 10/05/2022
+ms.localizationpriority: medium
+---
+
+# Activate your Outlook add-in on multiple messages (preview)
+
+With the item multi-select feature, your Outlook add-in can now activate and perform operations on multiple selected messages in one go. Certain operations, such as uploading messages to your Customer Relationship Management (CRM) system or categorizing numerous items, can now be easily completed with a single click.
+
+The following sections walk you through how to configure your add-in to retrieve the subject line of multiple messages in read mode.
+
+> [!IMPORTANT]
+> The item multi-select feature is only available in preview with a Microsoft 365 subscription in Outlook on Windows. Features in preview shouldn't be used in production add-ins. We invite you to test this feature in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
+
+## Prerequisites to preview item multi-select
+
+To preview the multi-select feature, install Outlook on Windows, starting with Version 2209 (Build 15629.20110). Once installed, join the [Office Insider program](https://insider.office.com/join/windows) and select the **Beta Channel** option to access Office beta builds.
+
+## Set up your environment
+
+Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator) to create an add-in project with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
+
+## Configure the manifest
+
+To enable your add-in to activate on multiple selected messages, you must add the [SupportsMultiSelect](/javascript/api/manifest/action?view=outlook-js-preview#supportsmultiselect&preserve-view=true) child element to the **\<Action\>** element and set its value to `true`. As item multi-select only supports messages at this time, the **\<ExtensionPoint\>** element's `xsi:type` attribute value must be set to `MessageReadCommandSurface` or `MessageComposeCommandSurface`.
```

Comment:
I think appending the second param breaks the bookmark/anchor... Here and other instances
```suggestion
To enable your add-in to activate on multiple selected messages, you must add the [SupportsMultiSelect](/javascript/api/manifest/action?view=outlook-js-preview&preserve-view=true#supportsmultiselect) child element to the **\<Action\>** element and set its value to `true`. As item multi-select only supports messages at this time, the **\<ExtensionPoint\>** element's `xsi:type` attribute value must be set to `MessageReadCommandSurface` or `MessageComposeCommandSurface`.
```